packages/booter-lb3app/src/lb3app.booter.ts

```diff
@@ -62,6 +63,19 @@ export class Lb3AppBooter implements Booter {
       this.mountRoutesOnly(lb3App, spec);
     }
 
+    const dataSources = lb3App.dataSources;
+    if (dataSources) {
+      const visited: DataSource[] = [];
+      Object.values(dataSources).forEach(ds => {
+        if (visited.includes(ds)) return;
+        visited.push(ds);
+        this.app
+          .bind(`datasources.lb3-${ds.name}`)
```

Comment:
On the second thought, I am not sure if it's a good idea to use `ds.name`. From what I remember, the code creating `ds.name` is messy and does not always honor the name provided by the user, replacing it with connector name sometimes.

Can we use the key in the `dataSource` object as the data-source name? As a nice side-effect, this will allow us to treat data-sources as opaque `object` type.

Not a big deal though, feel free to ignore this comment.